packages/features/bookings/lib/getLocationOptionsForSelect.ts

```diff
@@ -18,11 +19,11 @@ export default function getLocationsOptionsForSelect(
         return null;
       }
       const type = eventLocation.type;
+      const translatedLocation = getTranslatedLocation(location, eventLocation, t);
 
       return {
         // XYZ: is considered a namespace in i18next https://www.i18next.com/principles/namespaces and thus it get's cleaned up.
-        // Beacause there can be a URL in here, simply don't translate it if it starts with http: or https:. This would allow us to keep supporting namespaces if we plan to use them
-        label: locationString.search(/^https?:/) !== -1 ? locationString : t(locationString),
```

Comment:
This conditional was the reason why the inPerson address was not being displayed. The logic above states that we only check for URL's above and keep them as it is in the locationString variable and translate all the other values of locationString variable, hence that check will only work for the `type: link`. So if we have a location value like `{ type: "address", address: "King Street 1234, London" }` then our locationString value becomes King Street 1234, London and according to the above since this location doesn't contain `https:` it will try to translate it which will result in a blank string being returned since we don't have any value for this inside translations common.json file

Reply:
The `getTranslatedLocation` function that I'm using now to get the correctly translated location is already being used in the tooltip in one of the Booker components which can be found under the file `packages/features/bookings/components/event-meta/AvailableEventLocations.tsx` line 59